apertium/perceptron_spec.cc

```diff
@@ -92,9 +94,9 @@ PerceptronSpec::PerceptronSpec() {
       opcode_values[opcode_names[i]] = (Opcode)i;
     }
 
-    untagged_sentinel = make_sentinel_wordoids(L"!UNTAGGED!", L"!UT!");
-    token_wordoids_underflow = make_sentinel_token(L"!SURF_UNDERFLOW!", L"!TOK_UNDERFLOW!", L"!TUF!");
-    token_wordoids_overflow = make_sentinel_token(L"!SURF_OVERFLOW!", L"!TOK_OVERFLOW!", L"!TOF!");
+    untagged_sentinel = make_sentinel_wordoids("!UNTAGGED!"_u, "!UT!"_u);
+    token_wordoids_underflow = make_sentinel_token("!SURFNDERFLOW!"_u, "!TOKNDERFLOW!"_u, "!TUF!"_u);
```

Comment:
Did an errant search/replace hit `_U` here?

Reply:
looks like it